lib/plugins.ts

```diff
@@ -1,7 +1,15 @@
-var webdriver = require('selenium-webdriver');
-import {Logger} from './logger';
 import * as Q from 'q';
+import * as webdriver from 'selenium-webdriver';
+
 import {ConfigParser} from './configParser';
+import {Logger} from './logger';
+
+declare module 'q' {
```

Comment:
If this is a missing typing in `@types/q`, then this should be filed against DefinitelyTyped types-2.0 branch.  Also, a TODO should be added to remove this when the DefinitelyTyped PR is merged.